### ISSUE TITLE
.seek() returning Howler object

### ISSUE THREAD

**Issue by w-biggs** [thangngoc89/react-howler#72]
This is the code I'm working with:

```
import React, { Component } from 'react';
import { connect } from 'react-redux';
import { setPosition } from '../Store';
import ReactHowler from 'react-howler';

class Audio extends Component {
  player = React.createRef();

  componentDidMount(){
    window.requestAnimationFrame(this.getPosition);
  }

  componentDidUpdate(){
    if(this.props.audioPlaying){
      window.requestAnimationFrame(this.getPosition);
    }
  }

  getPosition = () => {
    console.log(this.player.howler.seek());
    //this.props.dispatch(setPosition(this.player.howler));
    if(this.props.audioPlaying){
      window.requestAnimationFrame(this.getPosition);
    }
  }

  render () {
    return (
      <ReactHowler ref={(ref) => (this.player = ref)} src="/assets/uniin.mp3" playing={this.props.audioPlaying} />
    );
  }
}

const mapStateToProps = (state) => ({ ...state.audioPlayer });

export default connect(mapStateToProps)(Audio);
```
Logged to the console is:
`Howl {_orientation: Array(3), _stereo: null, _pos: null, _pannerAttr: {…}, _onstereo: Array(0), …}`

Am I doing something wrong? I'm following the docs as far as I can tell...

**Comment by w-biggs** [comment]
It seems to work correctly if I set `html5={true}` on the component. Hm.

**Comment by fidellr** [comment]
yeah me too, sometimes i got Howler instance from `seek()` although i used `html5` props too

**Comment by Stenerson** [comment]
Hello,
Are you getting Howler objects returned right away or after some time?

It looks like Howler does return itself when calling `seek()` in a number of instances, most notably, if the sound is not loaded.

From: https://github.com/goldfire/howler.js/blob/9ad4eb1a973ca6faeef476315e39e938bf1547eb/src/howler.core.js#L1535

```javascript
      // If the sound hasn't loaded, add it to the load queue to seek when capable.
      if (self._state !== 'loaded' || self._playLock) {
        self._queue.push({
          event: 'seek',
          action: function() {
            self.seek.apply(self, args);
          }
        });

        return self;
      }
```

**Comment by SebastianGerS** [comment]
I'm experiencing the same problem but only after upgrading to v.3.7.4 of react-howler it does not seem to be an issue in 3.7.3

**Comment by Stenerson** [comment]
@SebastianGerS can you check if your version of Howler changed? The only change from `3.7.3` to `3.7.4` was removing the pinned version of Howler.

If you pin a previous version of Howler in your project does it fix the  issue?

It's probably also worth mentioning that this issue was opened before `3.7.4` was cut.

**Comment by SebastianGerS** [comment]
Ok so I looked in to it a bit more and it seam like it's happening in 3.7.3 aswell. 
I experienced it now after upgrading to 3.7.4 but when I think about it i probably did experience it when I first started to use it as well I just worked around it (but obviously not entirely). When downgrading to 3.7.3 the problem persists it's only that i'm not experiencing it in all circumstances. 

The problem seams to occur when I try to change src before the previous src has been loaded (and I'm  saving the position(seek) of howler in localstorage when the src is changed (as long as seek is not 0). 
I guess the easiest way for me to remedy this is just to also check that seek does not return a object before trying to save the position.

Anyway this seams to confirm what you are saying @Stenerson that seek returns the howler object when the sound is not loaded.
